src/helpers/constants.ts

```diff
@@ -36,3 +36,146 @@ export enum GENRE_IDS {
     WarPolitics = 10768, // tv only
     Western = 37,
 }
+
+export const COUNTRIES = [
+    { country: 'Andorra', abbrev: 'AD' },
+    { country: 'United Arab Emirates', abbrev: 'AE' },
+    { country: 'Antigua and Barbuda', abbrev: 'AG' },
+    { country: 'Albania', abbrev: 'AL' },
+    { country: 'Angola', abbrev: 'AO' },
+    { country: 'Argentina', abbrev: 'AR' },
+    { country: 'Austria', abbrev: 'AT' },
+    { country: 'Australia', abbrev: 'AU' },
+    { country: 'Azerbaijan', abbrev: 'AZ' },
+    { country: 'Bosnia and Herzegovina', abbrev: 'BA' },
+    { country: 'Barbados', abbrev: 'BB' },
+    { country: 'Belgium', abbrev: 'BE' },
+    { country: 'Burkina Faso', abbrev: 'BF' },
+    { country: 'Bulgaria', abbrev: 'BG' },
+    { country: 'Bahrain', abbrev: 'BH' },
+    { country: 'Bermuda', abbrev: 'BM' },
+    { country: 'Bolivia', abbrev: 'BO' },
+    { country: 'Brazil', abbrev: 'BR' },
+    { country: 'Bahamas', abbrev: 'BS' },
+    { country: 'Belarus', abbrev: 'BY' },
+    { country: 'Belize', abbrev: 'BZ' },
+    { country: 'Canada', abbrev: 'CA' },
+    { country: 'Congo', abbrev: 'CD' },
+    { country: 'Switzerland', abbrev: 'CH' },
+    // eslint-disable-next-line
```

Comment:
We should only disable the impacted rule here. I would double check what the rule is, but I think it is:
```suggestion
    // eslint-disable-next-line @prettier/prettier
```

future: It also wouldn't be a bad idea to create a ticket to fix this problem with our lint/prettier config so this comment can be removed. I believe there is a clash between the two configs.

Reply:
@Thenlie I was under the impression it was purely eslint with the following config:

```
"quotes": [
 "error",
 "single"
],
```

Where does prettier get involved?

Reply:
@maxh1231 When you change the outer set of quotes to double, es-lint throws an error due to the config you posted. When you fix that error with es-lint, prettier throws an error. Prettier wants you to use double quotes as the outer quotes if you have a single quote inside rather than escaping the single quote. I think we need to get prettier to allow the escaped character. 

For the disable comment here, we can do either `eslint-disable-next-line @prettier/prettier` or `eslint-disable-next-line quotes` depending on the state you leave it in.